### PR TITLE
Update widget-settings.md

### PR DESCRIPTION
**Pull request by srhc222**
Correction of PHP tags in the code snippets

### CODE REVIEW

**Comment by srhc222** [comment]
Correction of PHP tags in the code snippets

**Comment by rami-elementor** [comment]
@srhc222 Have you tested the code snippets?

**Comment by srhc222** [comment]
No, but I think it's a mistake.

‫בתאריך יום א׳, 13 באוג׳ 2023 ב-14:57 מאת ‪Rami Yushuvaev‬‏ <‪
***@***.***‬‏>:‬

File: src/widgets/widget-settings.md [hunk 1]
```diff
@@ -13,9 +13,9 @@ With JS templates we don’t need to retrieve the data using a special function,
 ```php
 <?php
```
Comment (rami-elementor) [inline src/widgets/widget-settings.md:14]: This is the opening tag.

File: src/widgets/widget-settings.md [hunk 2]
```diff
@@ -13,9 +13,9 @@ With JS templates we don’t need to retrieve the data using a special function,
...
 protected function content_template() {
-	?>
```
Comment (rami-elementor) [inline src/widgets/widget-settings.md:16]: This is the closing tag.

**Comment by rami-elementor** [comment]
Please review the PR again, if we change the code, the snippet will have two following PHP opening tags.

**Comment by srhc222** [comment]
Yes, it's fine.

‫בתאריך יום א׳, 13 באוג׳ 2023 ב-15:03 מאת ‪Rami Yushuvaev‬‏ <‪
***@***.***‬‏>:‬

**Pull request closed without merging**